Source/WebKit/UIProcess/API/Cocoa/_WKWebExtension.h

```diff
@@ -109,110 +109,122 @@ WK_CLASS_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))
  @param resourceBaseURL The directory URL to use for the new web extension.
  @param error Set to \c nil or an \c NSError instance if an error occurred.
  @result An initialized web extension, or `nil` if the object could not be initialized due to an error.
- @discussion This is a designated initializer. The URL must be a file URL that points
- to a directory containing a `manifest.json` file.
+ @discussion This is a designated initializer. The URL must be a file URL that points to a directory containing a `manifest.json` file.
  */
 - (nullable instancetype)initWithResourceBaseURL:(NSURL *)resourceBaseURL error:(NSError **)error NS_DESIGNATED_INITIALIZER;
 
-/*! @abstract The active errors for the extension. Returns `nil` if there are no errors. */
+/*!
+ @abstract The active errors for the extension.
+ @discussion This property returns an array of NSError objects if there are any errors, or `nil` if there are no errors.
+ */
 @property (nonatomic, nullable, readonly, copy) NSArray<NSError *> *errors;
 
-/*! @abstract The parsed manifest as a dictionary. Returns `nil` if the manifest was unable to be parsed. */
-@property (nonatomic, nullable, readonly, copy) NSDictionary<NSString *, id> *manifest;
+/*! @abstract The parsed manifest as a dictionary. */
+@property (nonatomic, readonly, copy) NSDictionary<NSString *, id> *manifest;
 
-/*! @abstract The parsed manifest version. */
+/*! @abstract The parsed manifest version, or `0` is if there is no version specified in the manifest.  */
```

Comment:
Is this allowed? Would that extension load?

Reply:
It is allowed. No the extension would not load. We would return `_WKWebExtensionErrorUnsupportedManifestVersion`. I'll note that.